DEPS

```diff
@@ -4,7 +4,7 @@ use_relative_paths = True
 
 deps = {
   "vendor/node": "https://github.com/brave/node.git@f51b9ab8ff446ca7b13be0de1bc12b854b23938d",
-  "vendor/native_mate": "https://github.com/zcbenz/native-mate.git@b5e5de626c6a57e44c7e6448d8bbaaac475d493c",
+  "vendor/native_mate": "https://github.com/zcbenz/native-mate.git@ad0fd825663932ee3fa29ff935dfec99933bdd8c",
```

Comment:
Yes- it's needed per the comment here:
https://github.com/brave/muon/pull/161#issuecomment-282778096